independent-publisher-connectors/Line Message/apiDefinition.swagger.json

```diff
@@ -0,0 +1,159 @@
+              "properties": {
+                "to": {
+                  "type": "string",
+                  "description": "user id"
+                },
+                "messages": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "type": {
+                        "type": "string",
+                        "description": "LINE message types",
+                        "default": "text",
+                        "title": "",
+                        "x-ms-visibility": "advanced",
+                        "enum": [
+                          "text",
+                          "sticker",
+                          "image",
+                          "location"
+                        ]
+                      },
+                      "text": {
+                        "type": "string",
+                        "title": "Text Type",
+                        "description": "hello world"
+                      },
+                      "packageId": {
+                        "type": "string",
+                        "title": "Sticker Type",
+                        "description": "Ex:446"
+                      },
+                      "stickerId": {
+                        "type": "string",
+                        "title": "Sticker Type",
+                        "description": "Ex:1988"
+                      },
+                      "originalContentUrl": {
+                        "type": "string",
+                        "title": "Image Type",
+                        "description": "https://example.com/image.jpg"
+                      },
+                      "previewImageUrl": {
+                        "type": "string",
+                        "title": "Image Type",
+                        "description": "https://example.com/image.jpg"
+                      },
+                      "title": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:My location"
+                      },
+                      "address": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:My location"
+                      },
+                      "latitude": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:35.687574,"
+                      },
+                      "longitude": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:139.72922,"
+                      }
+                    }
+                  },
+                  "description": "messages"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
@felaray it is not allowed to have a default response only. Please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
@Amjed-Ayoub I have already added the expected response. (200/400/401)